plutus-core/exe/Main.hs

```diff
@@ -121,7 +122,7 @@ data NormalizationMode = Required | NotRequired deriving (Show, Read)
 data TypecheckOptions = TypecheckOptions Input Format
 data EvalMode = CK | CEK deriving (Show, Read)
 data EvalOptions = EvalOptions Input EvalMode Format Timing
-data PrintMode = Classic | Debug deriving (Show, Read)
+data PrintMode = Classic | Debug | Readable | ReadableDebug deriving (Show, Read)
```

Comment:
I had this in the saturated builtins PR and decided to rescue it and put it in here.  It's not got much to do with this PR, but we might as well have it.